portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/administration/rolesandpermissions/cproles/RolesAdmin.testcase

```diff
@@ -108,6 +108,83 @@ definition {
 			value1 = "No roles were found.");
 	}
 
+	@description = "This test automates LPS-141903 and asserts that users are not counted in Roles after deactivating them"
+	@priority = "3"
+	test DeactivatedUsersAreNotCountedInRoles {
+		task ("Given two users assigned to corresponding sites as a Site Content Reviewer") {
+			JSONGroup.addGroup(groupName = "Site1");
+
+			JSONGroup.addGroup(groupName = "Site2");
+
+			JSONUser.addUser(
+				userEmailAddress = "user1@liferay.com",
+				userFirstName = "user1",
+				userLastName = "user1",
+				userScreenName = "user1");
+
+			JSONUser.addUser(
+				userEmailAddress = "user2@liferay.com",
+				userFirstName = "user2",
+				userLastName = "user2",
+				userScreenName = "user2");
+
+			JSONGroup.assignUserToGroup(
+				groupName = "Site1",
+				userEmailAddress = "user1@liferay.com");
+
+			JSONGroup.assignGroupRoleToUser(
+				groupName = "Site1",
+				roleTitle = "Site Content Reviewer",
+				userEmailAddress = "user1@liferay.com");
+
+			JSONGroup.assignUserToGroup(
+				groupName = "Site2",
+				userEmailAddress = "user2@liferay.com");
+
+			JSONGroup.assignGroupRoleToUser(
+				groupName = "Site2",
+				roleTitle = "Site Content Reviewer",
+				userEmailAddress = "user2@liferay.com");
+		}
+
+		task ("And given that this correctly displays that there are two users assigned as a Site Content Reviewer") {
+			Role.openRolesAdmin();
+
+			Navigator.gotoNavTab(navTab = "Site Roles");
+
+			LexiconList.viewListEntrySubtitle(
+				listEntry = "Site Content Reviewer",
+				listEntrySubtitle = "2 Assignees");
+		}
+
+		task ("When the first user is deactivated") {
+			User.openUsersAdmin();
+
+			User.singleDeactivateCP(userFirstName = "user1");
+		}
+
+		task ("Then that user should no longer be counted in Roles and no longer be listed as a member or the site") {
```

Comment:
The description for Then, "a member or the site" I believe the "**or**" should be "of" here, right? Pay attention to the singular and plural.